eclass/kernel-build.eclass

```diff
@@ -268,6 +313,26 @@ kernel-build_src_install() {
 kernel-build_pkg_postinst() {
 	kernel-install_pkg_postinst
 	savedconfig_pkg_postinst
+
+	if [[ -n ${KERNEL_IUSE_MODULES_SIGN} ]]; then
+		if use modules-sign && [[ -z "${MODULES_SIGN_KEY}" ]]; then
```

Comment:
Unnecessary quoting.

Reply:
Also fixed.

I somehow tend to automatically add quotes to my variables :P